Makefile

```diff
@@ -45,7 +45,7 @@ deps: tools
 .PHONY: test
 test: deps
 	@gocov test -race $(PKGS) | gocov-xml > coverage.xml
-	@gocov test github.com/newrelic/infra-integrations-sdk/jmx > /dev/null # TODO: fix race for jmx package
+	@gocov test github.com/newrelic/infra-integrations-sdk/v4/jmx > /dev/null # TODO: fix race for jmx package
```

Comment:
I just saw that this is fixed on https://github.com/newrelic/infra-integrations-sdk/pull/265/files
Once that is merged this can be rebased